### ISSUE TITLE
Can openi6x support TF Card?

### ISSUE THREAD

**Issue by suiaiyun** [OpenI6X/opentx#316]
Can openi6x support TF Card? It would be great if we could support it.

**Comment by Ultraexpert** [comment]
@suiaiyun i think as long as the library will fit.

**Comment by ajjjjjjjj** [comment]
I was experimenting on that and it requires too much FLASH memory. 
i6X have 4 times less FLASH memory than smallest officially supported devices, some things will never fit.

**Comment by suiaiyun** [comment]
> 

That's true. Now openi6x is great. It would be perfect if TF Card could be supported. Thank you for your great work to keep the i6x shining.
